src/styles/theme.scss

```diff
@@ -337,22 +411,40 @@ h5 {
   background-color: $primary-colour;
 }
 
+.card--alternate.card--with-icon {
```

Comment:
This is very specific and again deviation from the core. Why are card--alternate and card--with-icon being treated differently?

The card--with-icon class is for layout and the card--alternate is for color. I know in this design the alternate card includes a border but is it only the with icon one you need to target with that?

This class system was designed as single use rather than compound. So I think we are asking for trouble if we deviate too much from that.